qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CreateOffer.groovy

```diff
@@ -84,9 +55,10 @@ class CreateOffer implements CreateOfferInput, CalculatePrice{
             output.createdNewOffer(offer)
         } catch (DatabaseQueryException e) {
             output.failNotification(e.message)
-        } catch (Exception ignored) {
-            println ignored.message
-            println ignored.stackTrace.join("\n")
+        } catch (Exception unexpected) {
+            //TODO use logger facade instead of println
+            println unexpected.message
+            println unexpected.stackTrace.join("\n")
```

Comment:
You can use the logger facade now for this :)

Reply:
https://github.com/qbicsoftware/offer-manager-2-portlet/pull/284/files#diff-5d85ce4405375851f31753ee313c75aaf5ffb4d8b6770c52a156003c11b92499R59
True but not part of this PR